api/lib/resolvers.js

```diff
@@ -263,6 +244,23 @@ const resolvers = (networkList, notificationController) => ({
             validator.operatorAddress
           ].profile
         : undefined
+    },
+    feed: async (validator, _, { dataSources }) => {
+      const networkList = await db.getNetworks()
+      // get feed for this single validator
+      const validatorWithFeed = await getValidatorFeed(
```

Comment:
why is this returning the validator and the feed?

Reply:
Fixed